functions/exercise21/scriptSpec.js

```diff
@@ -1,15 +1,13 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
 /*globals getPossibleSubset*/
 
-describe("Getting all possible subset with a fixed length combinations in an array", function () {
-    it("If the array [1, 2, 3] we will get all subsets of length 2, we get subsets [2, 1], [3, 1], [3, 2], [3, 2, 1]", function () {
+describe("Getting all possible subset with a fixed length combinations in an array", (() => {
+    it("If the array [1, 2, 3] we will get all subsets of length 2, we get subsets [2, 1], [3, 1], [3, 2]", function () {
         var subsets = getPossibleSubset([1, 2, 3], 2);
-        expect(subsets).toEqual(
+        expect(subsets).toHaveSameItems([
```

Comment:
Please check if toHaveSameItems works as expected.
Please install jasmine collection matchers on the repository level.